x-pack/plugin/profiling/src/main/java/org/elasticsearch/xpack/profiling/action/SubGroup.java

```diff
@@ -19,7 +19,7 @@
 public class SubGroup implements ToXContentFragment {
     private final String name;
     private Long count;
-    @UpdateForV9(owner = UpdateForV9.Owner.PROFILING) // remove legacy XContent rendering
+    @UpdateForV10(owner = UpdateForV10.Owner.PROFILING) // remove legacy XContent rendering
```

Comment:
Unclear why XContent is legacy. The API isn't deprecated. The profiling team hasn't any further knowledge.